src/components/ChatRoom.jsx

```diff
@@ -10,18 +11,20 @@ function ChatRoom({ socket, userName }) {
   useEffect(() => {
     function addMessages(message) {
       let newMessages = messages.concat(message);
+      console.log('USE EFFECT newMessages : ',newMessages);
       updateMessage(newMessages);
     }
     socket.on(SocketEvents.LOBBY_MESSAGE, addMessages);
+    socket.on(SocketEvents.USER_MESSAGE, addMessages);
```

Comment:
probably don't need to add user message here! would just use the lobby message event already there. User message should be to just one user.